### ISSUE TITLE
Accessibility / a11y TalkBack descriptions

### ISSUE THREAD

**Issue by benvium** [sAleksovski/react-native-android-widget#79]
First up, thanks for this really interesting library! It's a challenging problem and your solution is neat.

Do you have any thoughts on how widgets using this could be made accessible with TalkBack? 
The ideal case would be that individual components could have `contentDescription` etc props. 
Failing that a reasonable fallback would be the ability to set a `contentDescription` on the widget root itself - that way a TalkBack user who taps anywhere on the widget could hear something like 'Music Widget. Currently playing all along the watchtower' etc.

For reference, at present when I tap my 'hello world' widget in TalkBack it reads out:

`Unlabelled, double-tap to activate labels available use tap with three fingers to view`

Thanks!

**Comment by sAleksovski** [comment]
The problem is that this library renders the widget to an image, so there are no Android views to add `contentDescription` to. 

I assume we can implement it in a similar way like the buttons, add an empty `FrameLayout` with `contentDescription` on the place of the components that have accessibility settings. 

I'll take a look when I have some free time.

**Comment by benvium** [comment]
Thanks for the response. Aha - yeah that sounds like a great plan,  FrameLayouts would very likely do the trick.

**Comment by sAleksovski** [comment]
I just released version `0.20.0` that has support for adding `accessibilityLabel` to the root widget, as well as all clickable area.

You can see more detailed docs at https://saleksovski.github.io/react-native-android-widget/docs/tutorial/make-widget-accessible , and try the example app by downloading the apk from https://github.com/sAleksovski/react-native-android-widget/releases/tag/v0.20.0

The Resizable Music widget and the List Widget Demo widgets have configured accessibility labels.

**Comment by benvium** [comment]
Great stuff, thank you!
